vllm/attention/backends/abstract.py

```diff
@@ -31,6 +31,10 @@ class AttentionType:
 
 class AttentionBackend(ABC):
     """Abstract class for attention backends."""
+    # For some attention backends, we allocate an output tensor before
+    # calling the custom op. When piecewise cudagraph is enabled, this
+    # makes sure the output tensor is allocated inside the cudagraph.
+    use_output: bool = False
```

Comment:
upon reflection, I think this name is not intuitive enough. can you rename it to `accept_output_buffer` to be clear?